ansible/packstack-demo.yml

```diff
@@ -14,7 +13,7 @@
     - passes.yml
   vars:
     # ssh key you want packstack to use to deploy
-    ssh_key: /home/jduncan/.ssh/id_rsa
+    ssh_key: "{{ lookup('env','HOME') }}/.ssh/id_rsa"
```

Comment:
Is there a reason we need to set this locally instead of picking up the group_var?